source/features/release-download-count.tsx

```diff
@@ -31,15 +36,15 @@ async function getAssetsForTag(tags: string[]): Promise<Tag> {
 	`);
 
 	const assets: Tag = {};
-	for (const [tag, release] of Object.entries(repository)) {
-		assets[tag] = (release as AnyObject).releaseAssets.nodes;
+	for (const [tag, release] of Object.entries(repository as ReleaseAssets[])) {
```

Comment:
This isn't right, this is why I don't like typing the API result too early. Repository is an object, not an array.